packages/profiling-node/bindings/cpu_profiler.cc

```diff
@@ -333,9 +333,8 @@ void SentryProfile::Start(Profiler *profiler) {
 
   // Initialize the CPU Profiler
   profiler->cpu_profiler->StartProfiling(
-      profile_title,
-      {v8::CpuProfilingMode::kCallerLineNumbers,
-       v8::CpuProfilingOptions::kNoSampleLimit, kSamplingInterval});
+      profile_title, v8::CpuProfilingMode::kCallerLineNumbers, true,
```

Comment:
Unclear why using the options signature causes this to sigabrt, but this will do too.